paddle/fluid/operators/math/sequence_padding.cc

```diff
@@ -59,6 +59,22 @@ void CopyValidData(framework::Tensor* dst_tensor,
   }
 }
 
+template <typename T>
+static void fast_mem_init(void* dest, size_t dest_size, const T* src,
+                          size_t num_bytes) {
+  if (dest == nullptr || dest_size == 0 || src == nullptr) return;
+
+  memcpy(dest, src, num_bytes);
```

Comment:
The origin is `pad_data[i] = *pad_value_data;`
Here is `pad_data[i] = pad_value_data;`?